custom_components/google_home/sensor.py

```diff
@@ -155,6 +171,55 @@ async def async_reboot_device(self) -> None:
 
         await self.client.reboot_google_device(device)
 
+    async def async_update_bluetooth(self) -> None:
+        """Service call to delete alarm on device"""
+        device = self.get_device()
+
+        if device is None:
+            _LOGGER.error("Device %s is not found.", self.device_name)
+            return
+
+        await self.client.request_bluetooth_scan(device=device)
+
+
+class GoogleHomeBTDevicesSensor(GoogleHomeBaseEntity):
+    """Google Home Alarms sensor."""
+
+    @property
+    def label(self) -> str:
+        """Label to use for name and unique id."""
+        return LABEL_BT_DEVICES
+
+    @property
+    def icon(self) -> str:
+        """Icon to use in the frontend."""
+        return ICON_BT_DEVICES
+
+    @property
+    def state(self) -> str | None:
+        device = self.get_device()
+        if not device:
+            return None
+        closest_device = device.get_closest_device()
+        return closest_device.mac_address if closest_device else STATE_UNAVAILABLE
```

Comment:
Wouldn't unknown be better? Home Assistant renders the entity as unavailable, which it isn't necessarily